CodeReview/src/overlays/CodeReviewCommentOverlay.cpp

```diff
@@ -43,6 +43,10 @@ CodeReviewCommentOverlay::CodeReviewCommentOverlay(Visualization::Item* associat
 	Super{{associatedItem}, style}
 {
 	setAcceptedMouseButtons(Qt::AllButtons);
+	// TODO what happens now in initializeForms() because of the item() function which takes commentInput_ as item storage?
+	// Does it not initialize the item if it is already initialized?
+	commentInput_ = new Visualization::Text{this, ""};
```

Comment:
This is the argument for the text, not style. Will the style be set correctly from initializeForms? Or would I need to do this here?